### PR TITLE
Delete connection

### PR DESCRIPTION
**Pull request by congwang09**
Allow LC to receive delete connection message and delete connection.

### CODE REVIEW

**Review by YufengXin — APPROVED** [review]
looks good. 
The next step is to test w/ Kytos

File: sdx_lc/messaging/topic_queue_consumer.py [hunk 1]
```diff
@@ -93,22 +93,37 @@ def handle_mq_msg(self, msg_body):
...
-                try:
-                    r = requests.post(str(OXP_CONNECTION_URL), json=msg_json)
-                    self.logger.info(f"Status from OXP: {r}")
-                except Exception as e:
-                    self.logger.error(f"Error on POST to {OXP_CONNECTION_URL}: {e}")
-                    self.logger.info(
-                        "Check your configuration and make sure OXP service is running."
-                    )
+                # send connection info to OXP
+                if msg_json["operation"] == "post":
```
Comment (sajith) [inline sdx_lc/messaging/topic_queue_consumer.py:109]: `msg_json.get("operation")` would be better here -- we'll crash if the message has no `operation` field.

Comment (sajith) [inline sdx_lc/messaging/topic_queue_consumer.py:109]:
```suggestion
                if msg_json.get("operation") == "post":
```

File: sdx_lc/messaging/topic_queue_consumer.py [hunk 2]
```diff
@@ -93,22 +93,37 @@ def handle_mq_msg(self, msg_body):
...
+                    try:
+                        r = requests.post(str(OXP_CONNECTION_URL), json=connection)
+                        self.logger.info(f"Status from OXP: {r}")
+                    except Exception as e:
+                        self.logger.error(f"Error on POST to {OXP_CONNECTION_URL}: {e}")
+                        self.logger.info(
+                            "Check your configuration and make sure OXP service is running."
+                        )
+                elif msg_json["operation"] == "delete":
```
Comment (sajith) [inline sdx_lc/messaging/topic_queue_consumer.py:118]:
```suggestion
                elif msg_json.get("operation") == "delete":
```

**Review by sajith — APPROVED** [review]

**Pull request merged**